### DISCUSSION TITLE
[Item] academic reading in social science

### DISCUSSION THREAD

**Discussion by Apriljzhang** [windingwind/zotero-better-notes#1148] [Note Templates]
### Description

academic reading in english

### Screenshots

<h1> <span>${topItem.getField("year")}_ ${topItem.getCreators().map((v)=>v.lastName).join("; ")}_${topItem.getField("title")} </span></h1>

<h2 style="color: #f411ab; background-color: #def2fd;" >💡 Info </h2>
<table> 
  <tr>
    <th style="background-color:#c9e4fa;">
      <p style="text-align: middle">🀄️</p>
    </th>
    <td style="background-color:#c9e4fa;"> ${topItem.getField("publicationTitle")}</td>  
  </tr>
  <tr>
    <th style="background-color:#def2fd;">
      <p class="FF" style="text-align: middle;">📰</p>
    </th>
    <td style="background-color:#def2fd;"> ${topItem.getCreators().map((v)=>v.lastName).join("; ")}, ${topItem.getField("year")}</td>  
  </tr>
  <tr>
    <th style="background-color:#c9e4fa;">
      <p style="text-align: middle">🤔 </p>
    </th> 
    <td style="background-color:#c9e4fa;"></td>  
  </tr>
 </table>
<h2 style="color: #E65100; background-color: #FFF8E1;" >❓ Gap and Research questions</h2>
<p></p>
<h2 style="color: #2E7D32; background-color: #F1F8E9;" >📝 Useful quotations</h2>
<p></p>
<h2 style="color: #3D405B; background-color: #F8EDEB;" >❗️ Relevant findings</h2>
<p></p>
<h2 style="color: #4A148C; background-color: #F5F5F5;" >🚩 Concepts defined</h2>
<p></p>
<h2 style="color: #1565C3; background-color: #E1F5FE;" >👨‍👩‍👦‍👦 Context and participants</h2>
<p></p>
<h2 style="color: #1565C0; background-color: #D4F2FF;" >🔬 Research Methods</h2>
<p></p>
<h2 style="color: #006064; background-color: #E0F7FA;" >⚙️ Speech pattern and words</h2>
<p></p>
<p></p>
<hr/>
<h3 style="color: #1565C0">Other：</h3>
<p></p>

### Test on Zotero version

7

### Test on Better Notes version

2.0.8

### Template Type

Item

### Template Share Code

```yaml
<h1> <span>${topItem.getField("year")}_ ${topItem.getCreators().map((v)=>v.lastName).join("; ")}_${topItem.getField("title")} </span></h1>

<h2 style="color: #f411ab; background-color: #def2fd;" >💡 Info </h2>
<table> 
  <tr>
    <th style="background-color:#c9e4fa;">
      <p style="text-align: middle">🀄️</p>
    </th>
    <td style="background-color:#c9e4fa;"> ${topItem.getField("publicationTitle")}</td>  
  </tr>
  <tr>
    <th style="background-color:#def2fd;">
      <p class="FF" style="text-align: middle;">📰</p>
    </th>
    <td style="background-color:#def2fd;"> ${topItem.getCreators().map((v)=>v.lastName).join("; ")}, ${topItem.getField("year")}</td>  
  </tr>
  <tr>
    <th style="background-color:#c9e4fa;">
      <p style="text-align: middle">🤔 </p>
    </th> 
    <td style="background-color:#c9e4fa;"></td>  
  </tr>
 </table>
<h2 style="color: #E65100; background-color: #FFF8E1;" >❓ Gap and Research questions</h2>
<p></p>
<h2 style="color: #2E7D32; background-color: #F1F8E9;" >📝 Useful quotations</h2>
<p></p>
<h2 style="color: #3D405B; background-color: #F8EDEB;" >❗️ Relevant findings</h2>
<p></p>
<h2 style="color: #4A148C; background-color: #F5F5F5;" >🚩 Concepts defined</h2>
<p></p>
<h2 style="color: #1565C3; background-color: #E1F5FE;" >👨‍👩‍👦‍👦 Context and participants</h2>
<p></p>
<h2 style="color: #1565C0; background-color: #D4F2FF;" >🔬 Research Methods</h2>
<p></p>
<h2 style="color: #006064; background-color: #E0F7FA;" >⚙️ Speech pattern and words</h2>
<p></p>
<p></p>
<hr/>
<h3 style="color: #1565C0">Other：</h3>
<p></p>
```

**Comment by windingwind** [comment]
Please always follow the instructions of the posting guidelines and use the template share code, not the template content. Thank you
